### ISSUE TITLE
Detect computer name and propose for the user

### ISSUE THREAD

**Issue by PrzemyslawKlys** [lithnet/access-manager#144]
**Is your feature request related to a problem? Please describe.**
It would be cool if AM would be able to detect the "user's" computer name so that it would provide an easy way for the user to get access to his computer. For SD agents it wouldn't be much help, but for developers that have limited number of machines or even just one it would make it easier.

**Comment by ryannewington** [comment]
Good idea! We're going to add a 'favorites' section to a future release. This will allow users to 'pin' their most used computers to their home screen for easy access.
